src/command.lisp

```diff
@@ -1057,7 +1057,8 @@ _~~A() {
         (dolist (line lines)
           (format stream "  ~A~%" line))
         (format stream "~%")
-        (format stream "    ~A~%" code)
+        (dolist (line (split-sequence #\Newline *s*))
```

Comment:
Where is `*s*` coming from? Perhaps that's a leftover from previous testing?

Reply:
Oops yeah, I just pushed a commit to fix that.